src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SNIProxy.cs

```diff
@@ -266,10 +266,8 @@ public uint WritePacket(SNIHandle handle, SNIPacket packet, bool sync)
         /// <summary>
         /// Reset a packet
         /// </summary>
-        /// <param name="handle">SNI handle</param>
-        /// <param name="write">true if packet is for write</param>
         /// <param name="packet">SNI packet</param>
-        public void PacketReset(SNIHandle handle, bool write, SNIPacket packet)
+        public void PacketReset(SNIPacket packet)
         {
             packet.Reset();
         }
```

Comment:
Looking at the calls to SNIPacket, I think the original idea was to have all operations to SNI internals like SNIPacket and SNIHandle to go through the SNIProxy. I wanted to leave it that way. 

@corivera do you have any preferences here?